src/components/MultiSelectGroup/MultiSelectGroup.stories.mdx

```diff
@@ -92,3 +92,15 @@ Note: when `layout = "column"` is passed, the MultiSelect components will have `
     <stories.MultiSelectGroupLayoutStory />
   </DSProvider>
 </Canvas>
+
+## `useMultiSelect` hook
+
+`useMultiSelect` is an optional hook provided by the component, for
```

Comment:
Please clarify `by the component`.  Which component?  `MultiSelect` or `MultiSelectGroup`?

Reply:
fixed